File: api-compositor/src/main/java/com/innteam/epicmarket/config/AppConfig.java
package com.innteam.epicmarket.config;

import com.innteam.epicmarket.product.ProductService;
import com.innteam.epicmarket.product.ProductServiceImpl;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class AppConfig {

    @Bean
    public ProductService productService() {
        return new ProductServiceImpl();
    }
}

File: file-storage/src/main/kotlin/ru/epicmarket/file/config/AppConfig.kt
package ru.epicmarket.file.config

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import ru.epicmarket.file.repository.FileRepository
import ru.epicmarket.file.service.FileService
import ru.epicmarket.file.service.FileServiceImpl

@Configuration
class AppConfig {

    @Bean
    fun fileService(repository: FileRepository) : FileService = FileServiceImpl(repository)
}